src/main/flight/rpm_filter.c

```diff
@@ -143,7 +143,7 @@ void rpmFilterInit(const rpmFilterConfig_t *config)
     const float loopIterationsPerUpdate = MIN_UPDATE_T / (pidLooptime * 1e-6f);
     numberFilters = getMotorCount() * (filters[0].harmonics + filters[1].harmonics);
     const float filtersPerLoopIteration = numberFilters / loopIterationsPerUpdate;
-    filterUpdatesPerIteration = rintf(filtersPerLoopIteration + 0.49f);
+    filterUpdatesPerIteration = lrintf(filtersPerLoopIteration + 0.49f);
```

Comment:
Isn't this (almost) `ceilf` ?

Reply:
Yeah, pretty weird. But let's keep it that way. There seems to be a small safe zone of 0.01 around numbers where it still floors them.

Reply:
Well, looking at it, design intent is probably to do filter updates in at most `loopIterationsPerUpdate`:
```
filterUpdatesPerIteration  = (numberFilters + loopIterationsPerUpdate - 1)  / loopIterationsPerUpdate;
```
But than MIN_UPDATE_T is actually MAX_UPDATE_T ..

Reply:
Sounds like it should actually be `floor` then to make sure it doesn't exceed `MAX_UPDATE_T`. Maybe it's "ceiled" to compensate for `MIN_UPDATE_T` actually being min and not max. I don't really know, does anybody want to try to improve documentation and simplify the magic? 😉

Reply:
Both variants make sense - either you wan update in bounded time (max/ceil) or you want to limit resource usage (min/floor) ...